package.json

```diff
@@ -94,15 +95,12 @@
   },
   "standard": {
     "parser": "@babel/eslint-parser",
-    "plugins": [
-      "eslint-plugin-compat"
-    ],
-    "extends": [
-      "plugin:compat/recommended"
-    ],
+    "plugins": ["eslint-plugin-compat"],
+    "extends": ["plugin:compat/recommended"],
     "env": {
       "browser": true
-    }
+    },
+    "ignore": ["**/spawn"]
```

Comment:
this folder was giving me this error
`Parsing error: No Babel config file detected for /Users/kevkevin/DEVDIR/stacker.news/spawn/capture.js. Either disable config file checking with requireConfigFile: false, or configure Babel so that it can find the config files.`